website/src/blogs/openebs-node-device-management-ndm-e2-80-8a--e2-80-8atroubleshooting-tips.md

```diff
@@ -0,0 +1,120 @@
+- Users or Operators like cStor Operator, Local PV provisioner interact with NDM by creating a BlockDeviceClaim CR. The BlockDeviceClaim will have properties like nodeName, required Capacity etc., The NDM operator will match these properties with the available BlockDevices and associate the one that matches all the requirements to BlockDeviceClaim.
+
+---
+
+### NDM Known Issues / Future Development Items
+
+- BlockDevices are not created for Partitions and LVM devices. If you need to use them, you have to manually create BlockDevice CR. The steps are mentioned in this [blog](https://blog.mayadata.io/openebs/creating-manual-blockdevice).
+
+OK. Let us get started with some common issues reported and how to troubleshoot them.
+
+---
+
+#### Scenario #1
+
+**BlockDevice CR is not created for a device available on my node.**
+
+I have some disks attached to the node. Installed openebs, but blockdevice resources are not created for the devices.
+
+**Symptom:** I have some disks attached to the node. Installed openebs, but blockdevice resources are not created for the devices.
+
+**Troubleshooting:**
+
+1. Check `lsblk` output of the node
+2. Get the NDM config map.
+3. Check if the mount point of the disk is excluded in the filter configurations in configmap.
+4. From lsblk output check if the blockdevice you want to use is an LVM/software raid/ partition/LUKS filesystem. NDM currently does not support these types.
+5. If none of the above works, the logs of NDM daemonset can be checked. It will have information of disk being detected , and at what point the disk was excluded from blockdevice creation, (like `excluded by path-filter`)
+
+    ![blockdevice creation](https://cdn-images-1.medium.com/max/800/0*q8rBQFw284gRYqjg)
+
+**Resolution:** Update the filter configuration in configmap and restart the NDM DaemonSet pod. This will create the blockdevices.
+
+---
+
+#### Scenario #2
+
+**After node reboot, one blockdevice became inactive and another blockdevice was created.**
+
+**Symptom:** When a node in the cluster rebooted. A blockdevice resource on that node was marked as inactive and a new resource was created. The new blockdevice also has the same details as the old one.
+
+**Troubleshooting:**
+
+1. Check `lsblk` output of the node
+2. Get the yaml of both blockdevices and compare them.
+3. Check to see `spec.Path` is different in both outputs.
+4. If yes, then the new blockdevice resource was created because the path changed
+5. Check if `kubernetes.io/hostname` is different, if yes , then the blockdevice was created because the hostname of the node changed.
```

Comment:
```suggestion
5. Check if `kubernetes.io/hostname` is different, if yes, then the blockdevice was created because the hostname of the node changed.
```